### PR TITLE
Initial app setup.

### PR DESCRIPTION
**Pull request by douglashall**
You can see the app running here: https://user-account.edxdev.org/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - lts/*
+cache:
+  directories:
+    - "~/.npm"
+before_install:
+  - npm install -g npm@latest
+  - npm install -g greenkeeper-lockfile@1.14.0
```
Comment (robrap) [inline .travis.yml]: Are we officially using greenkeeper and is there anything additional we need to do to configure?

File: config/webpack.prod.config.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader', // translates CSS into CommonJS
+            options: {
+              sourceMap: true,
+              minimize: true,
+            },
+          },
+          'postcss-loader',
```
Comment (robrap) [inline config/webpack.prod.config.js]: @staubina: FYI: This came from grades and doesn't exist in the cookie-cutter, but the footer won't work without this and there may be some browser issues.  You may not care for publisher, but it should be moved into the cookie-cutter.
@reillz10: Did you have plans to update the cookie-cutter or should we add a task?

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+node_modules
+npm-debug.log
+README.md
```
Comment (robrap) [inline .dockerignore]: Looks like you removed the README file altogether?  We should have one and translate it to rst (automated with pandoc).  Let me know if you want me to take care of that.

File: config/webpack.dev.config.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  // Specify additional processing or side-effects done on the Webpack output bundles as a whole.
+  plugins: [
+    // Generates an HTML file in the output directory.
+    new HtmlWebpackPlugin({
+      inject: true, // Appends script tags linking to the webpack bundles at the end of the body
+      template: path.resolve(__dirname, '../public/index.html'),
+    }),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: 'development',
+      BASE_URL: 'localhost:1994',
```
Comment (robrap) [inline config/webpack.dev.config.js]: We should update port 1994 to a unique port for eventual docker devstack integration.
Note: This port appears in multiple places, including the README when that is returned.

File: config/webpack.prod.config.js (hunk 2 above)
Comment (staubina) [reply config/webpack.prod.config.js]: I will try to keep this in mind.  We will eventually be adding the Header and Footer.

**Pull request merged**